app/services/iiif_print/manifest_builder_service_behavior.rb

```diff
@@ -71,27 +87,33 @@ def apply_v2_metadata_to_canvas(canvas:, presenter:)
 
     def sorted_canvases_v2(hash:, sort_field:)
       sort_field = Hyrax::Renderers::AttributeRenderer.new(sort_field, nil).label
-      hash["sequences"]&.first&.[]("canvases")&.sort_by! do |canvas|
-        selection = canvas["metadata"].select { |h| h["label"] == sort_field }
-        fallback = [{ label: sort_field, value: ['~'] }]
-        identifier_metadata = selection.presence || fallback
-        identifier_metadata.first["value"] if identifier_metadata.present?
+      hash['sequences']&.first&.[]('canvases')&.sort_by! do |canvas|
+        selection = canvas['metadata'].select { |h| h['label'] == sort_field }
+        fallback = [{ label: sort_field,
+                      value: ['~'] }]
+        sort_field_metadata = selection.presence || fallback
+        sort_field_metadata.first['value'] if sort_field_metadata.present?
       end
       hash
     end
 
-    def sorted_canvases_v3(hash:, **)
-      # TODO: flesh out metadata for v3
+    def sorted_canvases_v3(hash:, sort_field:)
+      sort_field = Hyrax::Renderers::AttributeRenderer.new(sort_field, nil).label
+      hash['items']&.sort_by! do |item|
+        selection = item['metadata'].select { |h| h['label'][I18n.locale.to_s] == [sort_field] }
+        fallback = [{ label: { "#{I18n.locale}": [sort_field] },
+                      value: { none: ['~'] } }]
+        sort_field_metadata = selection.presence || fallback
+        sort_field_metadata.first['value']['none'] if sort_field_metadata.present?
+      end
       hash
     end
 
-    def get_solr_docs(presenter)
+    def get_solr_hits(presenter)
       parent_id = presenter.id
       child_ids = presenter.try(:member_ids) || presenter.try(:ordered_ids)
```

Comment:
What happens when both tries fail?  We have `nil`.  Also, should we reverse the order of `presenter.try(:ordered_ids) || presenter.try(:member_ids)`

How about:

```ruby
ids = [presenter.id] + Array.wrap(presenter.try(:ordered_ids) || presenter.try(:member_ids))
```